akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -94,6 +96,7 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
   /**
    * Java API: roles that this member has
    */
+  @silent
```

Comment:
was this for not having `()`, could we add them given this is a java api?

Reply:
ah, no it was for being allowed to use `scala.collection.JavaConverters`